apps/sms/js/activity_handler.js

```diff
@@ -21,8 +21,6 @@ const ActivityDataType = {
 };
 
 var ActivityHandler = {
-  isLocked: false,
```

Comment:
Okay, let's make everyone happy here, we have old dedicated bug for this cleanup anyway - https://github.com/mozilla-b2g/gaia/pull/27394 :)